general-guides/community-wiki/general-troubleshooting.md

```diff
@@ -0,0 +1,112 @@
+
+* If you want to patch on a different device refer to the info in issue **#1** above.
+
+
+
+
+### **6. "App not installed."**
+
+If you encounter any sort of "App not installed" error during the installation of the patched app, follow these steps to resolve the issue:
+
+1. **Google Play Protect Warning:**
+   - If you see a Google Play Protect pop-up, tap **"More details"** and then **"Install anyway"**. Do not tap **"Got it"**, as this will cancel the installation. [View Image](https://imgur.com/a/Ck8nfhn).
+
+2. **Ensure Sufficient Storage:**
+    - Make sure your device has enough storage space.
+
+3. **Uninstall Existing Versions:**
+   - Remove any existing patched apps, including those in alternate user profiles or private folders.
+
+ > Youtube/YT Music
+> 
+>  - If you are trying to install the unpatched Youtube/YT Music APK from APKMirror.com, note that it is not necessary to install it directly. Instead, follow step 2 in the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners) to select and patch the APK correctly.
+
+
+4. **Check APK Compatibility:**
+   - Ensure that the patched APK matches the architecture of your device. Refer to step 1 in the [general guide](https://github.com/ReVanced-Extended-Community/Community-Guides/blob/main/general-guides/community-wiki/ytm-guide.md#1-downloading-rvx-manager-yt-music-apk--vanced-microg) for information on downloading the correct APK.
+
+> Youtube/YT Music
+>  - If you are trying to install the unpatched Youtube/YT Music APK from APKMirror.com, note that it is not necessary to install it directly. Instead, follow step 2 in the [beginners guide](https://www.reddit.com/r/revancedextended/comments/12vxggr/revanced_extended_guide_for_beginners) to select and patch the APK correctly.
+
+**Youtube/YT Music specific issues**
+
+5. **Patch with `GmsCore Support`:**
+   - Ensure that the `GmsCore Support` patch is applied when patching the APK.
+
+6. **GmsCore Version:**
+   - Ensure you are not installing an older version of GmsCore than the one already installed. Uninstall the current version via Settings > Apps > GmsCore > Uninstall.
+   - Make sure to download the latest GmsCore from [here](https://github.com/ReVanced/GmsCore/releases/latest)
+
+By following these steps, you should be able to resolve the majority of installation issues with Youtube/YT Music Extended.
+
+
+
+
+### **7. App crashes on startup**
+
+If the app is crashing on startup, there are 3 common causes of it:
+
+YouTube and YT Music:
+> **a)** You do not have GmsCore installed. Download and install [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).
+
+**b)** There was a failure or error while it was patching and you installed the broken APK. Refer to troubleshooting issue **#6** if this occurred.
+
```

Comment:
```suggestion
### **7. App crashes on startup**

If the app is crashing on startup, it may be because of one of the following:

1. There was a failure or error while it was patching and you installed the broken APK. Refer to troubleshooting issue **#5** if this occurred.

YouTube / YT Music:
> 2. You do not have GmsCore installed. Download and install [GmsCore](https://github.com/ReVanced/GmsCore/releases/latest).
>
> 3. The `GmsCore support` patch was not applied to the app.

YT Music:
> 4. The email you are signing in with has parental control restrictions.

```